server/fishtest/rundb.py

```diff
@@ -489,8 +489,7 @@ def get_finished_runs(
 
         count = self.runs.count_documents(q)
 
-        # Don't show runs that were deleted
-        runs_list = [run for run in c if not run.get("deleted")]
+        runs_list = [run for run in c]
```

Comment:
https://docs.python.org/3/library/functions.html#func-list
The built-in constructor `list()` is able to take an iterable, we could drop the comprehension.

```python
runs_list = list(c)
```